### PR TITLE
PSMDB-955 Updated the latest label in the version dropdown menu

### PR DESCRIPTION
**Pull request by nastena1606**
Now it points to the latest stable version correctly differentiating release candidates

Updated: source/netlify-conf/conf.py

### CODE REVIEW

File: source/conf-netlify/conf.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "4.2": "https://docs.percona.com/percona-server-for-mongodb/4.2/",
         "4.4": "https://docs.percona.com/percona-server-for-mongodb/4.4/",
         "5.0": "https://docs.percona.com/percona-server-for-mongodb/5.0/",
```
Comment (LenzGr) [inline source/conf-netlify/conf.py:28]: Should we then maybe change the 5.0 link as well?
```suggestion
        "5.0 (Release Candidate)": "https://docs.percona.com/percona-server-for-mongodb/5.0/",
```

Comment (nastena1606) [reply source/conf-netlify/conf.py:28]: As this label occupies 2 lines and looks ugly, it's decided to leave 5.0 label as is. @LenzGr please confirm

Comment (LenzGr) [reply source/conf-netlify/conf.py:28]: Fine with me!

**Review by LenzGr — APPROVED** [review]
LGTM, thanks for the clarification.

**Pull request merged**